pkg/scheduler/api/job_info.go

```diff
@@ -130,6 +131,10 @@ type TaskInfo struct {
 	NumaInfo   *TopologyInfo
 	PodVolumes *volumescheduling.PodVolumes
 	Pod        *v1.Pod
+
+	// CustomBindErrHandler is a custom callback func called when task bind err.
+	CustomBindErrHandler         func() error
+	CustomBindErrHandlerFinished bool
```

Comment:
please change `CustomBindErrHandlerFinished` to a better name.

Reply:
done for that.